coordinator/tests/services/translation/request/dispense/dispense-notification.spec.ts

```diff
@@ -17,19 +17,26 @@ import {
 } from "../../../../../src/services/translation/common/getResourcesOfType"
 import {ElementCompact} from "xml-js"
 import pino = require("pino")
-import {
-  createAgentPersonFromAuthenticatedUserDetailsAndPractitionerRole
-}from "../../../../../src/services/translation/request/agent-unattended"
+import {practitionerRoleDN} from "../../../../resources/test-data"
+import {convertOrganization} from "../../../../../src/services/translation/request/agent-unattended"
 
 const logger = pino()
+const mockCreateAuthorForDispenseNotification = jest.fn()
+const mockConvertOrganization = jest.fn()
+const mockCreateAgentPersonUsingPractitionerRoleAndOrganization = jest.fn()
 
 const actualMoment = requireActual("moment")
 jest.mock("moment", () => ({
   utc: (input?: MomentInput, format?: MomentFormatSpecification) =>
     actualMoment.utc(input || "2020-12-18T12:34:34Z", format)
 }))
```

Comment:
not related to your pr but this is some pretty heavyweight mocking here that im not sure is needed